pkg/spdk/disk.go

```diff
@@ -122,7 +122,7 @@ func (s *Server) DiskGet(ctx context.Context, req *spdkrpc.DiskGetRequest) (*spd
 	defer s.RUnlock()
 
 	// Check if the disk exists
-	bdevs, err := s.spdkClient.BdevAioGet(req.DiskName, 3000)
+	bdevs, err := s.spdkClient.BdevAioGet(req.DiskName, 0)
```

Comment:
What's the reason for better response time?

Reply:
The purpose of the timeout is waiting for a aio bdev with a given name to appear. For `DiskGet`, it is not necessary for the wait. If the bdev does not exist, it means the bdev is not created.